web-registry/src/components/organisms/Bridge/Bridge.tsx

```diff
@@ -61,8 +28,9 @@ export const Bridge = ({
           tabs={tabs}
           size={'xl'}
           sxs={tabsStyles.tabsInsideCard}
-          hideIndicator={!privateAccess}
+          hideIndicator={tabs?.length < 2}
```

Comment:
maybe it would be best to keep `privateAccess` so this is independent of the number of tabs, in case we want to add new tabs in the future

Reply:
made a `hideTabIndicator` option for the Bridge so this can be decided case-by-case, or removed later